trillian/ctfe/config.go

```diff
@@ -166,15 +167,17 @@ func ValidateLogConfig(cfg *configpb.LogConfig) (*ValidatedLogConfig, error) {
 	start, limit := cfg.NotAfterStart, cfg.NotAfterLimit
 	if start != nil {
 		vCfg.NotAfterStart = &time.Time{}
-		if *vCfg.NotAfterStart, err = ptypes.Timestamp(start); err != nil {
+		if err = start.CheckValid(); err != nil {
```

Comment:
`:=`

Reply:
Possibly remove the var declaration above and make them all ':='

Reply:
There was one place where the separate declaration was necessary, but I gave it its own declaration.